test/Unit/PowerShell/TestScripts/testBasicFunctionWithTriggerMetadata.ps1

```diff
@@ -8,4 +8,4 @@ param ($Req, $TriggerMetadata)
 # Used for logging tests
 Write-Verbose "a log"
 
-Push-OutputBinding -Name res -Value $TriggerMetadata.Req
+Push-OutputBinding -Name res -Value $Req
```

Comment:
This is no longer using the TriggerMetadata

Reply:
Will fix this.